src/pages/kubecon-europe-2020.js

```diff
@@ -0,0 +1,141 @@
+                variant={Button.VARIANT.PRIMARY}
+                href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_campaign=FY21-Q2-DEV-K8-EMEA-ELEV-OS-REG&utm_medium=OS&utm_source=ELEV&utm_content=REG&fiscal_year=FY21&quarter=Q2&gtm=DEV&program=K8&geo=EMEA"
+              >
+                Register here
+                <FeatherIcon
+                  className={styles.externalLinkIcon}
+                  name="external-link"
+                />
+              </Button>
+            </div>
+            <img
+              className={styles.img}
+              src={kubeconHeader}
+              alt="KubeCon and CloudNativeCon Europe 2020"
+            />
+          </section>
+          <section className={styles.section}>
+            <h2>Attend one of our lightning talks</h2>
+          </section>
+          <section className={cx(styles.section, styles.assetTable)}>
+            <div className={styles.point}>
+              <h4>We handle Prometheus, you keep Grafana</h4>
+              <p>Mon Aug 17 @ 14:35 CEST</p>
+              <p>Samuel Vandamme</p>
+            </div>
+            <div className={styles.point}>
+              <h4>How to use and customize Helm charts</h4>
+              <p>Mon Aug 17 @ 16:25 CEST</p>
+              <p>Douglas Camata</p>
+            </div>
+            <div className={styles.point}>
+              <h4>Kubernetes observability with context</h4>
+              <p>Tue Aug 18 @ 15:05 CEST</p>
+              <p>Stijn Polfliet</p>
+            </div>
+            <div className={styles.point}>
+              <h4>What is OpenTelemetry and how to get started?</h4>
+              <p>Tue Aug 18 @ 17:15 CEST</p>
+              <p>Lavanya Chockaligam</p>
+            </div>
+            <div className={styles.point}>
+              <h4>How to use and customize Helm charts</h4>
+              <p>Wed Aug 19 @ 15:05 CEST</p>
+              <p>Douglas Camata</p>
+            </div>
+            <div className={styles.point}>
+              <h4>OpenTelemetry Architecture</h4>
+              <p>Wed Aug 19 @ 16:25 CEST</p>
+              <p>John Watson</p>
+            </div>
+            <div className={styles.point}>
+              <h4>Kubernetes in the wild: best practices</h4>
+              <p>Thu Aug 20 @ 15:05 CEST</p>
+              <p>Martin Fuentes</p>
+            </div>
+            <div className={styles.point}>
+              <h4>Kubernetes observability with context</h4>
+              <p>Thu Aug 20 @ 16:50 CEST</p>
+              <p>Stijn Polfliet</p>
+            </div>
+
+            <div className={styles.highlightDiv}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="search"
+                size="2rem"
+              />
+              <h4 className={styles.highlightText}>
+                Go to the virtual expo tab, and find New Relic in Silver Hall B
+                to attend a lightning talk.
+              </h4>
```

Comment:
This is important info! I'd suggest putting it at the top of this section and using the callout component. See an example here: https://developer.newrelic.com/build-apps/add-time-picker-guide

![Screen Shot 2020-08-13 at 9 23 48 AM](https://user-images.githubusercontent.com/186715/90160531-bf96c600-dd46-11ea-8491-10c9ce9bf447.png)

Reply:
Done, added a note at the top of the section